src/main/java/org/folio/rest/impl/UserGroupAPI.java

```diff
@@ -224,7 +219,7 @@ public void deleteGroupsByGroupId(String groupId, String lang, Map<String, Strin
             .addField(ID_FIELD_NAME)
             .setJSONB(false)
             .setOperation("=")
-            .setValue("'"+ groupId +"'"));
+            .setVal(groupId));
```

Comment:
If there were a foreign key from users.patronGroup to userGroup.id we could replace this complete method by a PgUtil.deleteById invokation and avoid to have duplicated code.